debian/control

```diff
@@ -4,8 +4,9 @@ Priority: optional
 Homepage: http://ceph.com/
 Vcs-Git: git://github.com/ceph/ceph.git
 Vcs-Browser: https://github.com/ceph/ceph
-Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.hu>
-Uploaders: Sage Weil <sage@newdream.net>
+Maintainer: Ceph Maintainers <ceph-maintainers@lists.ceph.com>
```

Comment:
And to be clear, I'm not arguing at all.  I'm not sure what the convention is here.  Just because someone *could* do it years ago, does that mean they stay here forever?